packages/contracts/src/core/dao/DAO.sol

```diff
@@ -145,6 +147,7 @@ contract DAO is
         string calldata daoURI_
     ) external reinitializer(2) {
         _reentrancyStatus = _NOT_ENTERED; // added in v1.3.0
+        _registerInterface(type(ProtocolVersion).interfaceId); // added in v1.3.0
```

Comment:
```suggestion
        _registerInterface(type(IProtocolVersion).interfaceId); // added in v1.3.0
```

Moreove, I added `_reentrancyStatus` at the top so that it is the very first thing that is initialized.
The other additions of v1.3.0 don't necessarily need to be at the very top.
Maybe you can move it to/after the other registrations for clean code  purposes.